Source/DafnyCore/DooFile.cs

```diff
@@ -59,29 +61,38 @@ public class ManifestData {
     }
 
     public void Write(TextWriter writer) {
-      writer.Write(Toml.FromModel(this, new TomlModelOptions()).Replace("\r\n", "\n"));
+      var content = Toml.FromModel(this, new TomlModelOptions() {
```

Comment:
This looks like a good improvement to make, but does it mean that we're storing options that reference the file system in doo files after this change? I'll keep an eye out for which ones if so. It's been my intention for doo files to be portable and if such options need to be recorded in them we may need to look closer at alternatives for handling those options.

Independently of that, could this logic be in `TomlUtil` instead?

Reply:
I'll remove it.